samtranslator/model/__init__.py

```diff
@@ -536,6 +536,25 @@ def _construct_tag_list(
         # customer's knowledge.
         return get_tag_list(sam_tag) + get_tag_list(additional_tags) + get_tag_list(tags)
 
+    def propagate_tags(
+        self, propagate_tags: Optional[bool], resources: List[Resource], tags: Optional[Dict[str, Any]]
```

Comment:
Can be static or a separate resource.

Reply:
I think `propogate_tags` instead of a parameter can be `self.PropogateTags` because a function with an `Optional[bool]` without default value looks strange